fly/rc/targets_test.go

```diff
@@ -59,6 +59,30 @@ var _ = Describe("Targets", func() {
 	})
 
 	Describe("SaveTarget", func() {
+		Context("when managing .flyrc", func() {
+			It("creates any new file with 0600 permissions", func() {
+				err := rc.SaveTarget("foo", "url", false, "main", nil, "")
+				Expect(err).ToNot(HaveOccurred())
+				fi, statErr := os.Stat(flyrc)
+				Expect(statErr).To(BeNil())
+				Expect(fi.Mode().Perm()).To(Equal(os.FileMode(0600)))
+			})
+
+			Describe("when the file exists with 0755 permissions", func() {
+				BeforeEach(func() {
+					ioutil.WriteFile(flyrc, []byte{}, 0755)
```

Comment:
Super unlikely but I think it's still worth checking the `err` return value here. In the unlikely event that it fails it'll make the test failure much more likely. It'll also make our linter happier.

Reply:
Added this check.